### DISCUSSION TITLE
No Data for PH

### DISCUSSION THREAD

**Discussion by xomanuel** [electricitymaps/electricitymaps-contrib#4404] [Q&A] [answered]
Hi, I added data for the Philippines and requested another PR for the updated capacities. However, the live map still shows no data. How long does it take to display the data on live map?.

Thank you

**Comment by Kongkille** [comment, marked as answer]
Hi @xomanuel you can also read this thread here

https://github.com/electricitymap/electricitymap-contrib/discussions/2810

I am working on releasing contrib changes faster, ideally at least once a week.
